apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -9,58 +9,54 @@ import {
   getFooter,
   getFullPageById,
   getNavBar,
+  updateBlobById,
+  updatePageById,
 } from '../resource/resource.service'
 import { getSiteConfig } from '../site/site.service'
 
+// NOTE: Need to do validation like checking for existence of the page
```

Comment:
nit: maybe we should differentiate a note from a todo? Seems like they are used interchangeably here but might not be helpful in the future to know exactly what still needs to be done

Reply:
good catch, trouble you to make comments like this a non-nit!